Source/Utils/ScreenCover.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  ScreenCover.swift
+//  walmart
+//
+//  Created by David Sica on 5/9/16.
+//  Copyright © 2016 Walmart. All rights reserved.
+//
+
+import Foundation
+
+// Declaring conformance to this Protocol will whitelist the UIViewController's
+//  view.
+//
+//  Optionally, implement the shouldWhitelist method to determine whether to
+//  wholesale whitelist at runtime.  Or, declare to NOT whitelist but manually 
+//  hide sensitive information on the view.
+@objc protocol ScreenCoverProtocol {
+    optional func shouldWhitelist() -> Bool
```

Comment:
This is true, it is not currently used.  I added it only as a convenience for future consideration/implementation options.  Since it's optional, classes/interfaces can just declare.